### ISSUE TITLE
Route::livewire() is not works

### ISSUE THREAD

**Issue by slider23** [livewire/livewire#1159]
### Description

Hello Caleb, thanx for amazing new game-changing product.

But Route::livewire() is not works for me. )

### Steps to reproduce

- Install laravel
```
composer create-project --prefer-dist laravel/laravel livewire-demo
```

- Install livewire
```
composer require livewire/livewire
```

- Create component
```
php artisan livewire:make some-lw-component
```

- Add default livewire-style `resources/views/layouts/app.blade.php` 
```<html>
   <head>
   
       @livewireStyles
   </head>
   <body>
   
       @yield('content')
   
       @livewireScripts
   </body>
   </html>
```

- Add route to `routes/web.php`
```
use App\Http\Livewire\SomeLwComponent;
Route::livewire("/error-example", SomeLwComponent::class);
```
- Replace content in `resources/views/welcome.blade.php` by
```
@extends("layouts.app")
   
   @section("content")
       <p>Livewire component with blade macros works fine:</p>
       <p>@livewire("some-lw-component")</p>
       <p><a href="/error-example">Same component with Route::livewire("/error-example", SomeLwComponent::class); will throw exception</a></p>
   @endsection
```

- Enter in `/error-example` and get exception in LivewireManager.php:66: 
```
Livewire\Exceptions\ComponentNotFoundException 
Unable to find component: [App\Http\Livewire\SomeLwComponent]
``` 

Repo with this steps: https://github.com/slider23/livewire-router-bug-demo

Reason - getComponentClass() in LivewireManager want alias but receive name of component class from RouterMacros. 

**Context**
- Livewire version: 1.2.0
- Laravel version: 7.17.2
- Browser: Chrome

**Comment by slider23** [comment]
Maybe change find() in LivewireComponentsFinder like this:

```
	public function find($alias)
    {
        /**
         * probably $alias is really alias ?
         */
        if(isset($this->getManifest()[$alias])) return $this->getManifest()[$alias];

        /**
         * nope, $alias is classname
         */
        $rightAlias = null;
        foreach($this->getManifest() as $manifestAlias => $manifestClass){
            if($manifestClass === $alias) {
                $rightAlias = $manifestAlias;
                break;
            }
        }
        return $this->getManifest()[$rightAlias] ?? null;
    }
```

**Comment by dubcanada** [comment]
If you are using livewire as a component or part of an existing blade file you should just be using the regular routing/controller syntax, so

    Route::get("/whatever", 'SomeLwController@index');

And then you would `php artisan make:controller SomeLwController'

If you are using livewire as the whole page, you would use it such as

    Route::livewire("/error-example", 'some-lw-component');

And it would then load `SomeLwComponent.php` inside `app/Http/Livewire` rendering the view in the render function inside of the `layouts.app` template.

Basically how you have it setup is wrong, you are combining two different methods of implementation.

See https://laravel-livewire.com/docs/rendering-components for more details.

**Comment by slider23** [comment]
Oh, my mistake. I always use ::class notation for controllers in routes (Route::get("/whatever", [SomeLwController::class, "index"]);) and I was sure that I saw the same way in Livewire documentation for Livewire components. 

I was wrong, sorry for the inconvenience.
